analyzer/codechecker_analyzer/analysis_manager.py

```diff
@@ -335,6 +340,111 @@ def handle_failure(source_analyzer, rh, zip_file, result_base, actions_map):
         os.remove(plist_file)
 
 
+def kill_process_tree(parent_pid, recursive=False):
```

Comment:
I am thinking on that it would be good to mention in the function docstring that a same function can be found in the `web` part of the codechecker too so if somebody would like to change this function in the analyzer part it should be changed in the web part too. What do you think @gyorb?